process_manual/process_group_information.md

```diff
@@ -0,0 +1,211 @@
+Each skill group also had their own channel.
+Other than that, he had the following channels:
+- Announcements - only used for very important information that all should receive.
+- General - mostly used for relevant discussions and talks about the project, or for asking everyone for help.
+- Github_watch - github bot so you can follow all the sweet progress.
+- Flutter - general ressources and questions about Flutter.
+- Random - to keep everything meme-related away from the other channels
+
+We very strongly recommend that you add the setting that [displays people's full name instead of their obscure username](https://get.slack.help/hc/en-us/articles/115004686586-Show-members-full-names-in-your-workspace#show-full-names-1).
+
+We also recommended that you ask people to add their real name to GitHub, so you can search for that when assigning them, instead of having to deal with their GitHub usernames.
+They change that by going to <https://github.com/settings/profile> and changing their name to their real name.
+
+## General about meetings
+You will be responsible for planning most meetings.
+We created a Giraf Google Calendar that we shared with people, where we asked them to add all meetings all groups are welcome - that included skill group meetings and the general process meetings.
+When people add this calender to theirs, it wasn't necessary for us to promote the meeting on Slack, saving a lot of messages.
+If you choose to do the same, remember to add location to the meetings.
+
+We generally planned a retrospective on monday morning and sprint intro on tuesday morning, so you have time to review any changes to the process.
+This also gives the PO group time to organize their issues, and the development groups a bit of short time to catch up on their reports.
+
+Release preparation would usually be two full days.
+If the sprint ends on a friday, we held a release preparation from tuesday morning until friday at 13 o'clock. From 13 to 14 we held the release preparation.
+We had the release preparation early in the day, so as many as possible would attend it.
+
+## Our adaptation of GitFlow
+We used the [GitFlow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) as the branching strategy in the Giraf project.
+We also enforce squash merges from feature branches into the develop branch, so each feature will be in a single commit.
+
+This is our adaptation, but please read the entire section, as there might be better ways to do it.
+
+For the [release preparation](project_meetings.md#release-preparation), the process group creates a release branch, which is merged out from the master branch.
+Then commits are chosen from the develop branch.
+Only commits actually relevant for the upcoming release are added.
+This is called cherry picking commits.
+In the release preparation, developers will add bug-fixes to the release branch.
```

Comment:
This will be elaborated in the next section called "project meetings" and as such I left it out here. Should it be included anyway, since we have a reference to that section?

Reply:
Maybe add a ref?